.github/workflows/main-prd-deploy.yml

```diff
@@ -4,7 +4,7 @@ name: Prod Workflow
 on:
   push:
     tags:
-      - '2.[0-9]+.[0-9]+'
+      - '2.[0-9]+.[0-9]+-preprod'
       - '!*rc*'
```

Comment:
I think, I got this incorrect. We only need to use first condition. The 2nd condition i.e. `!*rc*` is not required and should be removed, otherwise due to OR condition, this pipeline will get (unintentionally) triggered on some other tag.

Reply:
fixed